src/components/Popup.js

```diff
@@ -3,6 +3,11 @@ import { closeButtons } from "../utils/constants.js";
 export default class Popup {
   constructor(popupSelector) {
     this._popupElement = document.querySelector(popupSelector);
+    this._submitButtonElement =
+      this._popupElement.querySelector(".modal__button");
+    this._saveButtonElement = this._popupElement.querySelector(
+      ".modal__loading-button"
+    );
```

Comment:
* `uploadingInfo` should be only in `PopupWithForm.js` because `Popup` is used for the image modal and it doesn't have any submit button

And `this._submitButtonElement` should be there as well